organizations/ajax_htmldata.php

```diff
@@ -579,8 +581,8 @@ function generateDowntimeHTML($downtime) {
 				<?php if ($externalLogin['loginURL']) { ?>
 				<tr>
 				<td style='vertical-align:top;text-align:left;'><?php echo _("Login URL:");?></td>
-				<td><?php echo $externalLogin['loginURL'];
-					if (strpos($externalLogin['loginURL'], 'http') !== 0) {
+				<td style="word-break: break-all;"><?php echo $externalLogin['loginURL'];
+					if (strpos($externalLogin['loginURL'], '://') !== 0) {
```

Comment:
This is wrong. We should add http:// in front of the URL if :// is not found. Should be:
`if (strpos($externalLogin['loginURL'], '://') === false) {`
(otherwise we end up with "http://ftp://")